test/BUILD

```diff
@@ -157,6 +164,13 @@ scala_library(
            glob(["src/main/scala/scala/test/mix_java_scala/*.java"]),
     jvm_flags = ["-Xms1G", "-Xmx4G"],
 )
+#Mix java scala
+scala_library(
+    name = "MixJavaScalaLibWithJvmFlags",
```

Comment:
@petemounce why is this target needed?
Asking for two reasons:
1. Nothing depends on it or runs it in the test_run.sh
2. The "existing" `MixJavaScalaLib` target also uses the `jvm_flags` attribute so other than the name I don't see any difference.

Did this push anything or conversely trying to protect from a regression? If so how is it different from the existing target?